components/breadcrumb-nav/css/_mixin.scss

```diff
@@ -13,10 +13,17 @@
 @mixin utrecht-breadcrumb-nav {
   --utrecht-focus-background-color: var(--utrecht-breadcrumb-nav-link-focus-background-color);
   --utrecht-link-background-color: var(--utrecht-breadcrumb-nav-link-background-color);
+  --utrecht-link-text-decoration: var(--utrecht-breadcrumb-nav-link-text-decoration);
   --utrecht-link-focus-color: var(--utrecht-breadcrumb-nav-link-focus-color, var(--utrecht-breadcrumb-nav-link-color));
-  --utrecht-link-focus-text-decoration: var(--utrecht-link-text-decoration);
+  --utrecht-link-focus-text-decoration: var(
+    --utrecht-breadcrumb-nav-link-focus-text-decoration,
+    var(--utrecht-breadcrumb-nav-link-text-decoration)
+  );
```

Comment:
Wasn't sure about this one and the `--utrecht-link-hover-text-decoration` change below. I'd have preferred hardcoding underline for this component, but that would ruin any customisability.

Reply:
Mentioned below too, but `disabled` already provides to option to limit the styles for the last item of the breadcrumb.